portfolio/src/main/webapp/gallery.html

```diff
@@ -1,15 +1,52 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Jared's Incredible Portfolio</title>
-    <link rel="stylesheet" href="gallery.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1><b>Hey there!!</b></h1>
-      <p>My name is Jared and this is my portfolio.</p>
-    </div>
-  </body>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Gallery</title>
+    <link rel="stylesheet" href="gallery.css"/>
+</head>
+<body>
+	<div class="main">
+		<h1>Little Jared</h1>
```

Comment:
It might be worth building and populating the gallery in JS later, since you could do things like select all the files in a folder, but this works for now. It would make the HTML less full and rely on fewer constants and special filenames.

Reply:
Yeah definitely, i'll have a look into it.